src/MyParcelComApi.php

```diff
@@ -1002,6 +1002,7 @@ public function deleteShipmentSurcharge(ShipmentSurchargeInterface $shipmentSurc
      */
     public function getDeliveryDates(
         string $serviceCode,
+        string $carrierCode,
```

Comment:
Same idea about reflecting the hierarchy:
```suggestion
    public function getDeliveryDates(
        string $carrierCode,
        string $serviceCode,
```